features/step_definitions/monitoring.rb

```diff
@@ -0,0 +1,13 @@
+Then(/^I check replicas of monitoring components for sno cluster$/) do
+
+  if infrastructure("cluster").infra_topology=="SingleReplica"
+
+    step %Q/there are 1 "daemonset" replicas in the "openshift-monitoring" project/
```

Comment:
@lihongyan1 I've update the helper PR to correct the grammar, so you should correct the call to something like 

```step %Q/there is 1 "daemonset" replica in the "openshift-monitoring" project/```

the same goes to the rest of the steps below.